### PR TITLE
Created new quiz tabs with mocked data

### PR DESCRIPTION
**Pull request by TSlashDreamy**
<details>
<summary>Preview</summary>

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/b0290a70-436c-4dd4-82c8-65d58430ffa5

</details>
<details>
<summary>Lint</summary>

зображення

</details>

### CODE REVIEW

File: src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  return (
+    <PageWrapper sx={styles.container}>
+      <Box component={ComponentEnum.Form} sx={styles.root}>
+        <AppTextField
+          InputLabelProps={styles.titleLabel}
+          InputProps={styles.titleInput}
+          fullWidth
+          inputProps={styles.input}
+          label={title ? '' : t('myResourcesPage.quizzes.defaultNewTitle')}
```
Comment (Tolik170) [inline src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx:48]: you can pass title to styles.titleLabel(title) and then in styles hide label,
visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,

File: src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          variant={TextFieldVariantEnum.Standard}
+        />
+        <Divider sx={styles.divider} />
+        <Box sx={styles.functionalButtons}>
+          <AppButton
+            size={SizeEnum.ExtraLarge}
+            variant={ButtonVariantEnum.Tonal}
+          >
+            {t('myResourcesPage.quizzes.createNewQuestion')}
+            <EditIcon fontSize='small' />
```
Comment (Tolik170) [inline src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx:72]: use SizeEnum

File: src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+          </AppButton>
+          <AppButton
+            size={SizeEnum.ExtraLarge}
+            variant={ButtonVariantEnum.Tonal}
+          >
+            {t('myResourcesPage.quizzes.addQuestion')}
+            <AddIcon fontSize='small' />
```
Comment (Tolik170) [inline src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx:79]: same here

File: src/pages/new-quiz/NewQuiz.constants.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { ReactElement } from 'react'
+
```
Comment (Tolik170) [inline src/pages/new-quiz/NewQuiz.constants.tsx:2]: remove space

File: src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  TextFieldVariantEnum
+} from '~/types'
+
+const EditQuizContainer = () => {
+  const { t } = useTranslation()
+  const [title, setTitle] = useState<string>('')
+  const [description, setDescription] = useState<string>('')
+
+  const handleTitleChange = (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
```
[the remark this answers is not in the document]
Comment (TSlashDreamy) [reply src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx]: fixed

File: src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx (hunk 1 above)
Comment (TSlashDreamy) [reply src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx]: > you can pass title to styles.titleLabel(title) and then in styles hide label, visibility: value ? VisibilityEnum.Hidden : VisibilityEnum.Visible,

thank you for your advice, slightly changed your approach and fixed it

File: src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx (hunk 2 above)
Comment (TSlashDreamy) [reply src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx]: > use SizeEnum

yup, fixed

File: src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx (hunk 3 above)
Comment (TSlashDreamy) [reply src/containers/my-quizzes/edit-quiz-container/EditQuizContainer.tsx]: > same here

fixed

File: src/pages/new-quiz/NewQuiz.constants.tsx (hunk 4 above)
Comment (TSlashDreamy) [reply src/pages/new-quiz/NewQuiz.constants.tsx]: > remove space

removed

**Review by ArturBekhDEV — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**